secretsmanager/secrets_test.go

```diff
@@ -129,7 +140,7 @@ func TestListSecretsWithFilter(t *testing.T) {
 		t.Errorf("expected %+v, got %+v", expected, out)
 	}
 
-	expected = []*secretsmanager.SecretListEntry{}
+	expected = []*string{}
 	out, err = s.ListSecretsWithFilter(context.TODO(), func(secret *secretsmanager.SecretListEntry) bool {
 		return false
 	})
```

Comment:
How about tests for showing and deleting secrets?

Reply:
😆